packages/aws-cdk/test/cdk-toolkit.test.ts

```diff
@@ -125,6 +125,7 @@ describe('deploy', () => {
         selector: { patterns: ['Test-Stack-A'] },
         requireApproval: RequireApproval.Never,
         hotswap: true,
+        logs: false,
```

Comment:
I think you can still make `logs` `false` by default in `CdkToolkit`, if you make them `true` by default in the CLI arguments.